### DISCUSSION TITLE
Help us name the 1.4 release!

### DISCUSSION THREAD

**Discussion by odette-elementl** [dagster-io/dagster#15266] [Community]
Hi everyone! Odette from the Dagster team here.

We'll be releasing Dagster v1.4 soon and want your feedback!

Every major and minor release gets named after a hit single or an album from a pop diva. Sometimes, the name of the song/album is inspired by what changes are coming with the release.

For inspiration, 1.4 will have:

- Auto-materialize evaluations in UI
- Improved performance for auto-materialize
- Asset backfill cancellation
- Op concurrency limits
- Among other things...

And here are some songs/albums we've previously named our versions after!

[Smooth Operator](https://www.youtube.com/watch?v=4TYv2PhG89A) (Sade) 
[Formation](https://www.youtube.com/watch?v=WDZJPJV__bQ) (Beyonce)
[Thank U, Next](https://www.youtube.com/watch?v=gl1aHhXnN1k) (Ariana Grande)
[Hello](https://www.youtube.com/watch?v=YQHsXMglC9A) (Adele)
[Cool for the Summer](https://www.youtube.com/watch?v=il9nqWw9W3Y) (Demi Lovato)
[Lucky Star](https://www.youtube.com/watch?v=ThHz9wlBeLU) (Madonna)
[The Edge of Glory](https://www.youtube.com/watch?v=QeWBS0JBNzQ) (Lady Gaga)
[Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) (Shakira)
[In the Zone](https://www.youtube.com/watch?v=we8h_2RFAyA) (Britney Spears)
[Waiting to Exhale](https://www.youtube.com/watch?v=_FT7ciAPuJk&list=PL_sN_rrP4od0gZ4C3LGkgcaRpVa_koQX4) (Whitney Houston, et al.)

[Listen to the full playlist](https://open.spotify.com/playlist/476UFttwzU8476ki1Aqcdd)

Drop a comment with your suggestions, and upvote the suggestions you like!

![image](https://github.com/dagster-io/dagster/assets/122291097/dde832b3-bba2-4596-814b-7bb016860cc3)

**Comment by aksestok** [comment]
[Material Girl](https://youtu.be/6p-lDYPR2P8) (Madonna)

**Comment by aksestok** [comment]
[All by myself](https://youtu.be/NGrLb6W5YOM) (Céline Dion)

**Comment by ipeterfulop** [comment]
Amaranth - Nightwish
